docs/wip/theme-shadowing.md

```diff
@@ -1,27 +1,28 @@
 # Shadowing
-This documentation will address Shadowing in Gatsby Themes focusing on how we use it in the Store Framework Jamstack (SFJ), you can find the Shadowing concept general explanations and examples in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/).
+This documentation will address Shadowing in Gatsby Themes focusing on how we use it in the Store Framework Jamstack (SFJ). You can find general explanations and examples of the Shadowing concept in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/).
 
 ## Importance of Shadowing
-In [Gatsby documentation](https://www.gatsbyjs.com/docs/conceptual/plugins-themes-and-starters/) we see that _"Gatsby themes allow Gatsby site functionality to be packaged as a standalone product for others (and yourself!) to easily reuse."_, this means that when you use the theme we developed you will be able to use pre-configured functionality, data sourcing, design and everything else we add to our Store Framework Theme.
+In [Gatsby documentation](https://www.gatsbyjs.com/docs/conceptual/plugins-themes-and-starters/) we see that _"Gatsby themes allow Gatsby site functionality to be packaged as a standalone product for others (and yourself!) to easily reuse."_, it means that when you use the theme we developed, you will be able to use pre-configured functionality, data sourcing, design and everything else we add to our theme.
```

Comment:
> Gatsby themes allow Gatsby site functionality to be packaged as a standalone product for others (and yourself!) to easily reuse."
 
According to [Gatsby documentation](https://www.gatsbyjs.com/docs/conceptual/plugins-themes-and-starters/), when using the theme we developed, you will be able to use pre-configured functionality, data sourcing, design and everything else that is part of it.